core/sticker_manager.py

```
@@ -159,20 +165,21 @@ async def add_sticker(self, file_bytes: bytes, original_filename: str, sticker_i
+        async with self._index_lock:
+            if sticker_id and str(sticker_id).strip():
+                sid = str(sticker_id).strip()
+                if sid in self._sticker_cache:
+                    raise ValueError(f"Sticker id {sid} already exists")
+                if sid.isdigit():
+                    try:
+                        numeric_id = int(sid)
+                        if numeric_id > self._sticker_index:
+                            self._sticker_index = numeric_id
+                    except Exception:
+                        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)